### ISSUE TITLE
Ppx deriving constructs currently have `[@@@coverage off]` 

### ISSUE THREAD

**Issue by mbarbin** [mbarbin/error-log#1]
Ppx deriving constructs are currently disabled via `[@@@coverage off]` due to a generated non visitable coverage point:

```ocaml
     let _ =
       fun (_ : t) ->
       ___bisect_visit___ 0;
       ()
     ;;
```
  
This is a known issue with work in progress to fix.

Current work around involves an `include T0` pattern, for example:

```ocaml
  module Mode = struct
    module T0 = struct
      [@@@coverage off]

      type t =
        | Quiet
        | Default
        | Verbose
        | Debug
      [@@deriving compare, equal, enumerate, sexp_of]
    end

    include T0
  end
```

**Comment by mbarbin** [comment]
Linking with https://github.com/ocaml-ppx/ppxlib/issues/473

**Comment by mbarbin** [comment]
Work in progress to fix in #5

**Comment by mbarbin** [comment]
Fixed by #6
